Client/Frontend/Home/HomepageFaviconImageViewModel.swift

```diff
@@ -5,7 +5,7 @@
 import Foundation
 import SiteImageView
 
-struct HomepageFaviconImageViewModel: FaviconImageViewModel {
+struct DefaultFaviconImageViewModel: FaviconImageViewModel {
```

Comment:
If it's the Default for all the app, we should probably relocate the file elsewhere. Also thinking since the corner radius comes from the HomepageViewModel we should also relocate that so it's more generic and not tied to the homepage.

Reply:
Oh and we should rename the file as well!

Reply:
I flip flopped on this one a little. The default values are the same for maybe 90% of cases it's used. I might actually just remove the protocol and create it with a default values in the init from within SiteImageView. That way the exceptions can still override it but we don't have the maintenance of creating a bunch of extra structs to replace one value.